tests/devices/test_default_qubit_jax.py

```diff
@@ -89,6 +89,26 @@ def circuit(x):
         # Just test that it works and spits our the right value.
         assert jnp.isclose(circuit(p), expected, atol=tol, rtol=0)
 
+    def test_qubit_circuit_different_interface(self, tol):
+        """Test that the device provides the correct
+        result for a simple circuit with a device using a different interface."""

```

Comment:
I am a bit confused. Do you mean "Test that a jax QNode returns the correct result when used with a non-jax device"? But would this test be in the right place then?

Reply:
This was an early integration test that I think is pretty well covered by  the tests in `test_qnode_jax`. I'll remove it.